sdk/core/azure-core/test/ut/retry_policy_test.cpp

```diff
@@ -140,6 +140,85 @@ class RetryPolicyTest_CustomRetry_False final : public RetryPolicy {
   {
     (void)response;
     (void)retryOptions;
+    return true;
```

Comment:
Does this compile if you leave off the `response` and `retryOptions` variable names? IIRC, it would allow you to avoid the void cast.

You can't do that for header functions because otherwise you can't document parameters, but in this case, I *think* it should work without them, and thus simplify the function.

As things currently stand, the `return true` gets lost in the `(void)foo` lines.